packages/suite-base/src/components/PlaybackControls/index.tsx

```diff
@@ -120,13 +130,23 @@ export default function PlaybackControls(props: {
   const togglePlayPause = useCallback(() => {
     if (isPlaying) {
       pause();
+
+      BroadcastLB.getInstance().postMessage({
+        type: "pause",
+        time: getTimeInfo().currentTime!,
+      });
     } else {
       const { startTime: start, endTime: end, currentTime: current } = getTimeInfo();
       // if we are at the end, we need to go back to start
       if (current && end && start && compare(current, end) >= 0) {
         seek(start);
       }
       play();
+
+      BroadcastLB.getInstance().postMessage({
+        type: "play",
+        time: getTimeInfo().currentTime!,
```

Comment:
You already have the currentTime from getTimeInfo()

```suggestion
        time: current!,
```

Reply:
True, I think it will also fix some instability when synchronizing instances on pause. Great catch, I will fix it.

Reply:
Done